acceptance-tests/src/test/java/me/atam/atam4jsampleapp/testsupport/Atam4jApplicationStarter.java

```diff
@@ -9,11 +9,12 @@
 public class Atam4jApplicationStarter {
 
     public static DropwizardTestSupport<ApplicationConfiguration> startApplicationWith(int initialDelayInMillis,
-                                                                                       Class testClass) {
+                                                                                       Class testClass, int periodInMillis) {
 
         ConfigOverride[] configOverrides = {
             ConfigOverride.config("testClasses", testClass.getName()),
-            ConfigOverride.config("initialDelayInMillis", String.valueOf(initialDelayInMillis))
+            ConfigOverride.config("initialDelayInMillis", String.valueOf(initialDelayInMillis)),
+            ConfigOverride.config("periodInMillis", String.valueOf(periodInMillis))
```

Comment:
Yes that might have been better in retrospect.  However, I'm not sure I understand why you'd do something just to reduce the impact - I think if it makes sense then don't worry about the impact... thoughts?